src/Queue/Connectors/RabbitMQConnector.php

```diff
@@ -69,11 +69,11 @@ public function connect(array $config): Queue
 
         if ($factory instanceof DelayStrategyAware) {
             /** @var DelayStrategy $delayStrategy */
-            $delayStrategyClass = Arr::get($config, 'delay.strategy', RabbitMqDlxDelayStrategy::class);
+            $delayStrategyClass = Arr::get($config, 'delay.strategy') ?? RabbitMqDlxDelayStrategy::class;
             $delayStrategy = new $delayStrategyClass();
 
             if ($delayStrategy instanceof BackoffStrategyAware) {
-                $backoffStrategyClass = Arr::get($config, 'delay.backoff.strategy', ConstantBackoffStrategy::class);
+                $backoffStrategyClass = Arr::get($config, 'delay.backoff.strategy') ?? ConstantBackoffStrategy::class;
```

Comment:
You can pass ConstantBackoffStrategy::class as a third argument to Arr::get

Reply:
Thanx for the feed back. 

I did that, but when the config item exists and is null, null will be returned. So that's wy i changed it to ??

Reply:
Third option in Arr::get is exactly for the default value.

Reply:
I know. But `Arr::get` returns the value if the key exists, even if it is set to `null`. When initiating a ne Class of `null` you get an error.

Because the config contains defaults and the key's exist, `null` is always returned when not using this feature.

Reply:
I updated the branch.

When...

1. The keys in the config are omitted. 
2. The values for the keys in the config are `null`.
3. The values for the keys in the config aren't existing classes with a correct interface.

...the default for that specific Awareness kicks in.

What ever happens in the config, the library will work without errors and is working like it was before (backwards compatible).